apps/site/lib/site_web/templates/event/_popup.html.eex

```diff
@@ -8,6 +8,7 @@
   id="<%= @id %>"
   aria-label="Event summary for <%= @event.title %>"
   aria-hidden="true"
+  data-js="enabled"
```

Comment:
I'm not sure this is needed since it won't be seen in the `.no-js` situation anyway?

Reply:
Correct; removing it as well 😅